website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,57 @@ Think of this file as where you [bootstrap the settings or configuration of your
 [terraform](/core-concepts/components/terraform), then [this is where](/cli/configuration/components#terraform-component-behavior)
 you'd specify the command to run (e.g. [`opentofu`](/core-concepts/projects/configuration/opentofu)),
 the base path location of the components, and so forth.
+## Base Path
 
-## Configuration File (`atmos.yaml`)
+The base path defines the root directory for components, stacks, workflows, and Atmos configurations. It can be set using the `ATMOS_BASE_PATH` environment variable or by passing the `--base-path` command-line argument. Both absolute and relative paths are supported. When a relative path is provided, it will be expanded to an absolute path relative to the current working directory, as Atmos executes commands that may change the working directory (e.g., Terraform).
+
+When the `base_path` is explicitly provided, if it is not an absolute path, then it is resolved to the absolute path relative to the current working directory.
```

Comment:
```suggestion
When the `base_path` is explicitly defined, it will always be resolved to the absolute path relative to the current working directory, if an absolute path was not already set.
```